File: a2sv/number-of-times-binary-string-is-prefix-aligned.py
class Solution:
    def numTimesAllBlue(self, flips: List[int]) -> int:
        _max = float('-inf')
        count = 0
        for i in range(len(flips)):
            _max = max(_max , flips[i])
            if i+1 == _max:
                count +=1
        return count
        

File: a2sv/number-of-times-binary-string-is-prefix-aligned.java
class Solution {
    public int numTimesAllBlue(int[] flips) {
        int _max = -1;
        int count = 0;
        for (int i = 0; i < flips.length; i ++){
            _max = Math.max(_max, flips[i]);
            if (i+1 == _max){
                count++;
            }
        }
        return count;
        
    }
}